projects/isy-angular-widgets/src/lib/hauptfenster/hauptfenster.component.html

```diff
@@ -56,7 +56,8 @@
 
   <div class="grid grid-nogutter flex-grow-1">
     @if (showLinksnavigation) {
-      <div
+      <aside
+        aria-label="linksnavigation"
```

Comment:
Translation should be used here. As otherwise screen readers only read out in German